robot/impl/resource_manager.go

```diff
@@ -742,12 +748,17 @@ func (manager *resourceManager) completeConfigForRemotes(ctx context.Context, lr
 					return
 				}
 
-				// Trigger completeConfig goroutine execution when a change in remote
-				// is detected.
+				// Attempt to trigger completeConfig goroutine execution when a change in remote
+				// is detected, but does not block if triggerConfig is full.
 				select {
 				case <-lr.closeContext.Done():
 					return
 				case lr.triggerConfig <- struct{}{}:
+				default:
+					rr.Logger().Debugw(
+						"remote attempted to trigger reconfiguration, but there is already one queued.",
+						"remote", remConf.Name,
```

Comment:
[nit] Do you need to include remote name if we're already logging through the remote's client? Also, can we use `CDebugw`?

Reply:
updated